commerce-api/src/main/java/com/liferay/commerce/payment/method/CommercePaymentMethod.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.payment.method;
+
+import aQute.bnd.annotation.ProviderType;
+
+/**
+ * @author Luca Pellizzon
+ */
+@ProviderType
+public interface CommercePaymentMethod {
+
+	public String getName();
+
+	public int getPaymentType();
+
+	public String pay();
```

Comment:
@luca-pellizzon What this pay stay for?

Reply:
@marco-leo just a placeholder for the payment method, will be substitute with processPayment() or something similar